### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. 09.01.1

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#25171]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear the chat messages read out once, and not have to hear the entire message read (Speaker, message, timestamp) at once, then have to hear it a second time when I arrow through the message, before I hear the next message.

## Details

Message containers have an `aria-labelledby` attribute applied that points to the inner container of the actual message. This causes screen readers to read out the entire message when the outer container is selected, and forces users to listen to that message a second time before they can listen to the next message. Screenshot attached below.

## Acceptance Criteria

- [ ] Messages are read out once, no duplicated content

## Environment

* https://staging.va.gov/virtual-agent/

## Proposed Solution (if known)
* Removing the `aria-labelledby` attributes from message containers should clear the issue. Manual testing with screen readers will confirm this behavior.

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

## Screenshots or Trace Logs

![Screen Shot 2021-05-24 at 2 29 18 PM](https://user-images.githubusercontent.com/934879/119400671-1cdd8180-bca0-11eb-8751-edf4e058b2b4.png)

**Comment by ianpsant** [comment]
Requires Microsoft change. Would require assistance in filing Microsoft ticket.

**Comment by ianpsant** [comment]
Filed Microsoft bug mentioned above.

**Comment by noahgelman** [comment]
This doesn't have a 1-to-1 success criteria since it's more of a general bug.

**Comment by allison0034** [comment]
This ticket has been reassigned from CCIssue00.00 "No VA.gov Experience Standard for the issue found" to CCIssue09.01 "Markup and meta data". No VFS action is needed.

**Comment by ianpsant** [comment]
Resolved on Microsoft ticket https://github.com/microsoft/BotFramework-WebChat/issues/3937.
